localgovdrupal/localgov_alert_banner
# removes polyfills and es6 code

Closes #378 

## What does this change?

Removes polyfill files and ES6 specific files, so we can just use modern JS instead.

## How to test

Create some alert banners, make sure they still work

---
Thanks to [Big Blue Door](https://www.bigbluedoor.net/) for sponsoring my time to work on this.
